lib/tasks/demo_seed.rake

```diff
@@ -246,6 +246,17 @@ namespace :demo do
     end
     UserRole.grant(user_staff, UserRole::FACILITY_STAFF, facility)
 
+    user_facility_administrator = User.find_by(username: "mfa123@example.com")
+    unless user_facility_administrator
+      user_facility_administrator = User.new(username: "mfa123@example.com",
+                                             email: "mfa123@example.com",
+                                             first_name: "Macy",
+                                             last_name: "Administator")
```

Comment:
Can we say "Facility Administrator" to make it clear from the name that it's a facility admin, and not a global admin?